site/components/payment-streams/EndTime.js

```diff
@@ -22,32 +23,37 @@ const EndTime = function ({
     year: 'numeric'
   })
   return (
-    <fieldset className="my-2">
-      <legend className="text-gray-600 font-bold">{t('for-how-long')}</legend>
-      <select
-        onChange={e => onYearsChange(parseInt(e.target.value, 10))}
-        value={years}
-      >
-        {[...Array(11).keys()].map(year => (
-          <option key={year} value={year}>{`${year} years`}</option>
-        ))}
-      </select>
-      <select
-        onChange={e => onDaysChange(parseInt(e.target.value, 10))}
-        value={days}
-      >
-        {[...Array(365).keys()].map(day => (
-          <option key={day} value={day}>{`${day} days`}</option>
-        ))}
-      </select>
-      <select
-        onChange={e => onHoursChange(parseInt(e.target.value, 10))}
-        value={hours}
-      >
-        {[...Array(24).keys()].map(hour => (
-          <option key={hour} value={hour}>{`${hour} hours`}</option>
-        ))}
-      </select>
+    <fieldset className="my-2 w-full">
+      <InputTitle>{t('for-how-long')}</InputTitle>
+      <div className="flex space-x-2">
+        <select
+          className="p-2 w-1/3 rounded-xl"
+          onChange={e => onYearsChange(parseInt(e.target.value, 10))}
+          value={years}
+        >
+          {[...Array(11).keys()].map(year => (
+            <option key={year} value={year}>{`${year} years`}</option>
+          ))}
+        </select>
+        <select
+          className="p-2 w-1/3 rounded-xl"
+          onChange={e => onDaysChange(parseInt(e.target.value, 10))}
+          value={days}
+        >
+          {[...Array(365).keys()].map(day => (
+            <option key={day} value={day}>{`${day} days`}</option>
+          ))}
+        </select>
+        <select
+          className="p-2 w-1/3 rounded-xl"
+          onChange={e => onHoursChange(parseInt(e.target.value, 10))}
+          value={hours}
+        >
+          {[...Array(24).keys()].map(hour => (
+            <option key={hour} value={hour}>{`${hour} hours`}</option>
+          ))}
+        </select>
```

Comment:
These `option` tags are not translated... If you can, maybe we can fix them upstream (I am ok if you prefer to create an issue in the PF repo and fix it later)

Reply:
I opened a separated PR with this: https://github.com/hemilabs/pure.finance/pull/3